stress/shell/src/main/java/alluxio/stress/cli/GetPinnedFileIdsBench.java

```diff
@@ -129,4 +131,34 @@ public GetPinnedFileIdsParameters getParameters() {
   public static void main(String[] args) {
     mainInternal(args, new GetPinnedFileIdsBench());
   }
+
+  /**
+   * Helper client that is used to minimizes the deserialization overhead by calling
+   * getPinnedFileIdsCount instead of getPinnedFileIdsList
+   */
+  private static class PinListFileSystemMasterClient
+      extends alluxio.worker.file.FileSystemMasterClient {
+    private static final Logger LOG =
+        LoggerFactory.getLogger(PinListFileSystemMasterClient.class);
+    private FileSystemMasterWorkerServiceGrpc.FileSystemMasterWorkerServiceBlockingStub mClient =
+        null;
+
+    public PinListFileSystemMasterClient(MasterClientContext conf) {
+      super(conf);
+    }
+
+    @Override
+    protected void afterConnect() throws IOException {
+      mClient = FileSystemMasterWorkerServiceGrpc.newBlockingStub(mChannel);
+    }
+
+    public int getPinListLength() throws IOException {
```

Comment:
I'm still not convinced this method should be implemented here. Why not call FileSystemMasterClient.getPinlist() on the call site and call `size()` on it? That way we don't have to complicate the code by subclassing.

Reply:
I added some comments on why I'm doing this in 6fb2ad27114d5689b74afbc20e9e325ebecd9322, which you might have missed. Could you take a look at the comments of this class?